cogs/Moderation.py

```diff
@@ -150,7 +153,7 @@ async def lock(self, msg, *args):
     # For the reaction add event regarding assigning roles, check RoleMenu cog
     @commands.Cog.listener()
     async def on_raw_reaction_add(self, reaction):
-        if reaction.member.bot: # Ignore reaction remove and add events from itself (when editing the menu)
+        if reaction.member.bot and not self.config.processReaction: # Ignore reaction remove and add events from itself
```

Comment:
FIgure out how much of this can be pulled out to a function, probably best to call a function directly from Tasks rather than triggering an event